src/BlockSourcePage.tsx

```diff
@@ -107,7 +107,7 @@ export default (): React.ReactElement => {
         </WaitForData>
       </SingleCardPage>
       {blockStatus === 'ok' && blockData instanceof AstarteCustomBlock && (
-        <Row className="justify-content-end m-3">
+        <div className="d-flex flex-column flex-md-row justify-content-end gap-3 m-3">
```

Comment:
This is a wealthy amount of classes to achieve right alignment + wrapping, which is not really readable for new users.
Since we use this pretty much for most page buttons, what about wrapping this in a component?

Reply:
The complexity is inherent and in the end it's just CSS, so it cannot be easily removed.

Abstracting this kind of layout for buttons, which tends to repeat, can be an idea. However we are working on a redesign of Astarte Dashboard, so IMO it makes sense to avoid abstracting until we know the path we're headed to: I would leave this as is.